src/components/Comments/RemoveComment.jsx

```diff
@@ -0,0 +1,48 @@
+import Button from "@mui/material/Button";
+import CircularProgress from "@mui/material/CircularProgress";
+import Box from "@mui/material/Box";
+import { removeComment } from "../api";
+import { useState } from "react";
+
+export default function RemoveComment({
+  commentId,
+  setRenderToggle,
+  setNumberofComments,
+}) {
+  const [loading, setLoading] = useState(false);
+
+  const handleClick = () => {
```

Comment:
It took me a bit longer to figure out what button this was for because of the generic name - try be specific with what your functions are called, making sure it relates to what they do